pkg/payloads/v1alpha1/clusters.go

```diff
@@ -37,10 +37,10 @@ const (
 	Scanned ScanStatusType = "scanned"
 )
 
+// +k8s:deepcopy-gen=true
```

Comment:
Why generate `DeepCopy` functions for types of `payloads` package?

I think it's not necessary since these types are not used in `clientset` or controller-runtime clients.

Reply:
> Why generate DeepCopy functions for types of payloads package?

To prevent side effects.

> I think it's not necessary since these types are not used in clientset or controller-runtime clients.

Those are not the only places where this'd be used.

Reply:
So where are the `DeepCopy` functions being used for `payload` package types?

Reply:
On the Saas server.